nunit.templates/ItemTemplates/Test/NUnit Setup Fixture/SetupFixture.vb

```diff
@@ -0,0 +1,25 @@
+﻿Option Explicit On
+
+Imports System
+Imports NUnit.Framework
+
+Namespace $rootnamespace$
+
+    <SetUpFixture>
+    Public Class $safeitemname$
+
+        <OneTimeSetUp>
+        Public Sub OneTimeSetUp()
+            ' TODO Add code here that is run before
+            '  all tests in the assembly are run
```

Comment:
Extra whitespace after `'`. Could you fit this all on a single line?

\+ 1 more